### PR TITLE
Decoded TESObjectCell further

### PR DESCRIPTION
**Pull request by TommInfinite**

### CODE REVIEW

**Comment by ThirdEyeSqueegee** [comment]
Please fix the build errors and ensure that the PR name follows the conventional commits pattern (see other PRs for examples)

**Review by ThirdEyeSqueegee — CHANGES_REQUESTED** [review]
Please fix the build errors and ensure that the PR name follows the conventional commits pattern (see other PRs for examples)

**Comment by Qudix** [comment]
New headers need to be added to `RE/Starfield.h` in alphabetical order

**Comment by ThirdEyeSqueegee** [comment]
> New headers need to be added to `RE/Starfield.h` in alphabetical order

There's a script that adds them automatically during build configuration

File: CommonLibSF/include/RE/T/TESObjectCELL.h [hunk 1]
```diff
@@ -12,7 +13,8 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
-
-		//
+    	UInt16					flags;				// 48
```
Comment (gottyduke) [inline CommonLibSF/include/RE/T/TESObjectCELL.h:16]: Use `std::uint16_t` in CommonLib. `UInt16` is undefined

**Pull request closed without merging**